types/ember-data/index.d.ts

```diff
@@ -917,12 +917,12 @@ declare module 'ember-data' {
              */
             snapshots(): any[];
         }
-        class Snapshot<K extends keyof ModelRegistry = ''> {
+        class Snapshot<K extends keyof ModelRegistry = any> {
```

Comment:
This use of `any` means that this type checks:

```ts
let s: DS.Snapshot;
```

But this does not, if there is no `Waffle` model registered:

```ts
let s: DS.Snapshot<'waffle'>;
```